packages/editor/package.json

```diff
@@ -39,14 +39,18 @@
     "apollo-utilities": "^1.3.2",
     "array-move": "^2.2.1",
     "core-js": "^3.4.8",
+    "css-loader": "^4.2.1",
     "express": "^4.17.1",
+    "file-loader": "^6.0.0",
     "graphql": "^14.5.8",
     "graphql-tag": "^2.10.1",
+    "leaflet": "^1.6.0",
     "nanoid": "^2.1.7",
     "pretty-bytes": "5.3.0",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
+    "react": "^16.13.1",
```

Comment:
Why the update of react?

Reply:
When I added react and react-dom via yarn add (I wasn't aware that it's already there) this must have happened.